src/styles/_help.scss

```diff
@@ -0,0 +1,11 @@
+.questions-section {
+  margin-top: 0;
+}
+
+div.questions-section > ol > li:nth-child(4) > ol > li {
```

Comment:
What is this? If a new question is added won't this break?

Reply:
the lists are not occupying the space as the unordered lists do, removing this to be consistent with others.

Reply:
@greeshmareddyg this rule targets a single list item in a single ordered list that is the fourth list item of an ordered list inside a specific class.

This rule is far too specific to be useful. The slightest change would break it.

Reply:
![image](https://user-images.githubusercontent.com/21186867/76885554-ca136900-6855-11ea-97f9-e6a5373acfe1.png)

here the inner list items not occupying the width as ol, how can we fix this, what selectors can I use to fix that.

Reply:
@greeshmareddyg there's nothing special about this list right? There's no reason it _shouldn't_ be styled correctly. Which means you should make an issue on the theme rather than trying to style such as specific thing.

A better selector would just be

```
.questions-section li li {
	// styles here
}
```

This targets all list items that are children of other list items. So if someone put items under item 2 or 5 down the road, those would _also_ get the styles. As opposed to just the 4th item.